test/test.tools.js

```diff
@@ -50,37 +60,29 @@ describe("Monkeypatching", () => {
 
 describe("Test tools for parsing new gmail body_params", () => {
     const gmail = new Gmail();
-    const data =  JSON.parse(testData.new_gmail_archive_action_body_params);
-
-    it("get thread id", () => {
-        const thread = gmail.tools.get_thread_id(data);
```

Comment:
This test is not related to event-parsing, but covers some primitives which event-parsing relies on, so I think this should stay.